hashintel/hash
# Update HASH welcome message

## 🌟 What is the purpose of this PR?



This PR updates the welcome copy of the HASH app

## 🔗 Related links





- [Update HASH welcome message](https://app.asana.com/0/1202250835094696/1202381444789113) _(internal)_


## 🔍 What does this change?




- Updates the welcome message users see when logged into the app

## 📜 Does this require a change to the docs?



- No


## ❓ How to test this?

1.  Checkout the branch
1.  Login. The welcome page should match attached screenshot

## 📹 Demo


![Screenshot 2022-06-06 at 10 22 02](https://user-images.githubusercontent.com/16928688/172140035-2f518a67-0532-4d8b-912c-6ef271407f76.png)
